python/cugraph/dask/mg_bfs/bfs.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from dask.distributed import wait, default_client
+from cugraph.dask.common.input_utils import get_local_data
+from cugraph.mg.traversal import mg_bfs_wrapper as mg_bfs
+import cugraph.comms.comms as Comms
+
+
+def call_bfs(sID, data, local_data, start, return_distances):
+    wid = Comms.get_worker_id(sID)
+    handle = Comms.get_handle(sID)
+    return mg_bfs.mg_bfs(data[0],
+                         local_data,
+                         wid,
+                         handle,
+                         start,
+                         return_distances)
+
+
+def bfs(input_graph,
+        start,
+        return_distances=False):
+
+    """
+    Find the PageRank values for each vertex in a graph using multiple GPUs.
+    cuGraph computes an approximation of the Pagerank using the power method.
+    The input graph must contain edge list as  dask-cudf dataframe with
+    one partition per GPU.
+
+    Parameters
+    ----------
+    edge_list : dask_cudf.DataFrame
+        Contain the connectivity information as an edge list.
+        Source 'src' and destination 'dst' columns must be of type 'int32'.
+        Edge weights are not used for this algorithm.
+        Indices must be in the range [0, V-1], where V is the global number
+        of vertices.
+    start : Integer
+        The index of the graph vertex from which the traversal begins
+
+    return_sp_counter : bool, optional, default=False
+        Indicates if shortest path counters should be returned
+
+    Returns
+    -------
+    df : cudf.DataFrame
+        df['vertex'][i] gives the vertex id of the i'th vertex
+
+        df['distance'][i] gives the path distance for the i'th vertex from the
```

Comment:
What happens to `df['distance'][i]`  when `return_distances=False`? this should be documented.